### DISCUSSION TITLE
Lock symbol at some tag fields?

### DISCUSSION THREAD

**Discussion by ionlyaskforeden** [quodlibet/quodlibet#4632] [Q&A] [answered]
Hello everyone,

could someone tell me what these locks mean? I suppose the lock stands for read only correct?
Is this some sort of DRM or what am i looking at?

![image](https://github.com/user-attachments/assets/ef779d2c-a530-4678-a751-7c06a858f15d)

**Comment by declension** [comment, marked as answer]
Yes this just means they are read-only, i.e. cannot be edited (over _all_ the files selected). 

Typically this will be because some of those files are an mp4 or other format with fixed tags defined, rather than freeform (though #323 etc)
